### PR TITLE
Add a statistic for "how much content has been released yet"

### PR DESCRIPTION
**Pull request by yoe**

### CODE REVIEW

File: ansible/playbooks/roles/encoder-master/files/sql_exporter/sreview.collector.yml [hunk 1]
```diff
@@ -11,3 +11,12 @@ metrics:
   - count
   query: |
     SELECT state, count(*) FROM talks GROUP BY state;
+- metric_name: timereleased
```
Comment (SuperQ) [inline ansible/playbooks/roles/encoder-master/files/sql_exporter/sreview.collector.yml]:
```suggestion
- metric_name: video_released_seconds
```

File: ansible/playbooks/roles/encoder-master/files/sql_exporter/sreview.collector.yml [hunk 2]
```diff
@@ -11,3 +11,12 @@ metrics:
...
+  type: gauge
+  help: 'Length of all the released videos added together'
+  key_labels:
+  - label
+  values:
+  - total_time
+  query: |
+    select 'video_released_seconds' as label, extract(epoch from sum((endtime - starttime)::interval)) as total_time from talks where state = 'done';
```
Comment (SuperQ) [inline ansible/playbooks/roles/encoder-master/files/sql_exporter/sreview.collector.yml]:
```suggestion
    select extract(epoch from sum((endtime - starttime)::interval)) as total_time from talks where state = 'done';
```

File: ansible/playbooks/roles/encoder-master/files/sql_exporter/sreview.collector.yml [hunk 3]
```diff
@@ -11,3 +11,12 @@ metrics:
   - count
   query: |
     SELECT state, count(*) FROM talks GROUP BY state;
+- metric_name: timereleased
+  type: gauge
+  help: 'Length of all the released videos added together'
+  key_labels:
+  - label
```
Comment (SuperQ) [inline ansible/playbooks/roles/encoder-master/files/sql_exporter/sreview.collector.yml]: Drop this, we don't need any labels here.

File: ansible/playbooks/roles/encoder-master/files/sql_exporter/sreview.collector.yml (hunk 1 above)
Comment (yoe) [reply ansible/playbooks/roles/encoder-master/files/sql_exporter/sreview.collector.yml]: done

**Comment by yoe** [comment]
Should all be fine now.

**Pull request merged**